### PR TITLE
Use yarn repo on all scripts

### PR DESCRIPTION
**Pull request by nemsia**
* Add yarn to app_dependencies
* Remove jessie backports config file

### CODE REVIEW

**Comment by anmol26s** [comment]
I don't know if repos is better way of installing yarn. @Jibec did the yarn installation code for PeerTube.
If he thinks its ok then we can merge it into testing. What about the people who already have PeerTube and Mastodon installed. Can it break the installation some how if this is implemented?

**Comment by nemsia** [comment]
This package isn't very important, just needed on install or upgrade but i think it's better to use ynh_install_app_dependencies to install this dependencie, @Jibec stay this on this work on mastodon_ynh: https://github.com/YunoHost-Apps/mastodon_ynh/commit/18b69b2fdfbd1178334b5ea5a3c9e75fb439a10f

File: scripts/remove [hunk 1]
```diff
@@ -47,11 +47,9 @@ fi
 ynh_remove_app_dependencies
 ynh_remove_nodejs
 
-# Remove yarn
-sudo apt-get remove --purge -y yarn
-
-# Delete backport from source.list 
+# Delete backport and yarn from source.list 
 ynh_secure_remove /etc/apt/sources.list.d/jessie-backports.list
+ynh_secure_remove /etc/apt/sources.list.d/yarn.list
```
Comment (Jibec) [inline scripts/remove]: I'm unsure we should remove it without being sure no other application uses this package. I assume it's the same for jessie-backports.

**Comment by Jibec** [comment]
I think we can do even better and ask peertube dev to release compiled files so we don't have to compile it ourselves. Let's try this :)

File: scripts/remove (hunk 1 above)
Comment (nemsia) [reply scripts/remove]: Mostodon_ynh re-add it on upgrade, no idea if that is the best solution.

**Comment by Jibec** [comment]
OK, that's not so bad then. Let's keep it as it.
As peertube can't get rid of yarn, I'll give it a try this weekend. Including upgrade.
What tests did you do so far?
-- 
Jean-Baptiste Holcroft

**Comment by anmol26s** [comment]
Its working for me. For uniform version of the yarn, this is a better solution.

**Comment by Jibec** [comment]
I do confirm it works fine on both debian 8 and debian 9, x86_64, on clean install and upgrade, thank you :+1:

**Comment by anmol26s** [comment]
@Jibec Thanks for the your testing. If you are ok  with the PR, then we can merge this?

**Comment by Jibec** [comment]
Yep, I'm ok with this PR.

**Pull request merged**